docs/pages/components/ActionMenu.mdx

```diff
@@ -0,0 +1,424 @@
+---
+title: ActionMenu
+description: A dropdown menu component that displays an ActionList when with actions.
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/ActionMenu/ActionMenu.tsx
+---
+
+import { useState, useRef } from 'react'
+import { 
+  ActionMenu, 
+  ActionList, 
+  ActionListItem, 
+  ActionListGroup, 
+  ActionListSeparator,
+  ActionListLinkItem,
+  Button, 
+  IconButton, 
+  Icon,
+  AnchoredOverlay 
+} from '@deque/cauldron-react'
+
+```jsx
+import { 
+  ActionMenu, 
+  ActionList, 
+  ActionListItem 
+} from '@deque/cauldron-react'
+```
+
+## Overview
+
+The `ActionMenu` component is a dropdown menu that displays an `ActionList` when triggered. It displays a list of items that are typically contextual actions to perform operations or select single or multiple contextual states.
+
+## Examples
+
+### Basic Usage
+
+A simple action menu with a button trigger:
+
+```jsx example
+<ActionMenu
+  trigger={
+    <Button variant="secondary">
+      Actions
+      <Icon type="chevron-down" />
```

Comment:
What do you think about the basic example also showing the chevron changing direction when opened? I'm not seeing that in Figma, but I assume that's what we want if someone were to use it. The chevron changing direction is also in the example here. https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions-active-descendant/